python/graphstorm/model/gnn.py

```diff
@@ -914,6 +936,8 @@ def get_input_embeds(input_nodes):
                                                 edge_mask=edge_mask,
                                                 target_ntypes=infer_ntypes,
                                                 task_tracker=task_tracker)
+        # Called when model.eval()
+        embeddings = model.normalize_node_embs(embeddings)
```

Comment:
you only call this function in this branch?

Reply:
We should call it for all branches.

Reply:
udpated